indexing-service/src/main/java/org/apache/druid/indexing/common/config/TaskConfig.java

```diff
@@ -131,11 +167,11 @@ public TaskConfig(
       @JsonProperty("ignoreTimestampSpecForDruidInputSource") boolean ignoreTimestampSpecForDruidInputSource,
       @JsonProperty("batchMemoryMappedIndex") boolean batchMemoryMappedIndex, // deprecated, only set to true to fall back to older behavior
       @JsonProperty("batchProcessingMode") String batchProcessingMode,
-      @JsonProperty("storeEmptyColumns") @Nullable Boolean storeEmptyColumns
+      @JsonProperty("storeEmptyColumns") @Nullable Boolean storeEmptyColumns,
+      @JsonProperty("baseTaskDirLocations") @Nullable List<StorageLocationConfig> baseTaskDirLocations
```

Comment:
Do we need to add this new parameter to the configuration document?

Reply:
Yes. I'll do that as well